wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/monitoring/ClusterTopologyMonitorImpl.java

```diff
@@ -653,14 +653,9 @@ protected String getSuggestedWriterNodeId(final Connection connection) throws SQ
 
     final HashMap<String, HostSpec> hostMap = new HashMap<>();
 
-    int expectedColumnCount = 4;
```

Comment:
Expecting a column count of 4 was causing failovers against multi-az clusters because the multi-az topology query expects a different number of columns (3)